ansible/roles/littlesis/tasks/passenger.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Add Passenger key
+  apt_key:
+    keyserver: 'hkp://keyserver.ubuntu.com:80'
+    id: 561F9B9CAC40B2F7
+
+- name: Add passenger Repo
+  apt_repository:
+    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger xenial main'
+    state: present
+    update_cache: yes
+
+- name: Install Passenger & Nginx
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - nginx-extras
+    - passenger
+
+
+- name: Copy static files
+  synchronize:
+    src: ../static/
+    dest: /var/www/static
+    delete: yes
+    recursive: yes
+  tags: static
+
+- name: Set permissions on static files
+  file:
+    path: /var/www/static
+    state: directory
+    owner: root
+    group: root
+    mode: 0644
+    recurse: yes
+  tags: static
+    
+- name: Enable passenger in nginx.conf
+  lineinfile:
+    regexp: 'include /etc/nginx/passenger.conf;'
```

Comment:
might be *slightly* clearer for future users (or future you) to note somehow that the point of this bit is to remove a commented-out line.